nsxt/resource_nsxt_policy_gateway_dns_forwarder_test.go

```diff
@@ -248,7 +248,7 @@ resource "nsxt_policy_gateway_dns_forwarder" "test" {
 %s
   display_name = "%s"
   description  = "%s"
-  gateway_path = nsxt_policy_tier%d_gateway.test.path
+  gateway_path = nsxt_policy_tier % d_gateway.test.path
```

Comment:
Spaces here break this test - this is not an HCL % operator but a sprintf %d.
There are a few of these below.

Reply:
Resolved in https://github.com/vmware/terraform-provider-nsxt/commit/c4d727ee3f2865e81de7c333d72ac570948b9abc.